Fixed center of mass bug to take into account objects

josephbriones opened the pull request:
Modified the formula for how center of mass was being calculated. Called focus on center of mass function after algorithm is instantiated.

This PR resolves #16.

josephbriones committed:
Fixed center of mass bug to take into account objects

+ modified the formula for how center of mass was being calculated. Called focus on center of mass function after algorithm is instantiated.

jdaymude commented:
Fixes #16.

jdaymude commented on res/qml/main.qml:
```diff
@@ -274,6 +274,7 @@ ApplicationWindow {
       onClicked: {
         vis.forceActiveFocus()
         instantiate(algorithmSelectBox.currentText)
+        vis.focusOnCenterOfMass();
```
I would remove this "fix". It adds the annoyance of resetting the camera to center even if the user has already figured out some view they like more (e.g., offset a bit to the left to move it out from under the sidebar).

jdaymude commented on ui/visitem.cpp:
```diff
@@ -39,18 +39,23 @@ void VisItem::focusOnCenterOfMass() {
   }
 
-  view.setFocusPos(sum / numNodes);
+  for(const Object* t: system->getObjects()) {
```
Very minor detail, but why `t` instead of, say, `obj`?

jdaymude reviewed (changes requested):
See comments for minor fixes.

josephbriones committed:
Minor bug fixes

- removed focus on center of mass upon initialization.
~ renamed object variable
